src/Module/Login.php

```diff
@@ -140,8 +140,8 @@ private static function passwordAuthentication($username, $password, $remember)
 			}
 		} catch (Exception $e) {
 			logger('authenticate: failed login attempt: ' . notags($username) . ' from IP ' . $_SERVER['REMOTE_ADDR']);
-			notice($e->getMessage() . EOL);
-			goaway(self::getApp()->get_baseurl() . '/login');
+			info('Login failed. Please check your credentials.' . EOL);
+			goaway();
```

Comment:
This empty parameter in ```goaway()``` will most likely create a notice because of a missing parameter.